### PR TITLE
Fix `WavProperties` reading

### LINKED ISSUE #32: Hound's test samples not working.
I tried test samples from [hound](https://github.com/ruuda/hound) and none of it was read successfully using examples/tag_reader.

Here is log from terminal:
```
------------Processing /home/user/Prejemi/hound-master/testsamples/nonstandard-01.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/nonstandard-02.wav file..
thread 'main' panicked at 'Error: Failed to read file!: Wav("Extensible format identified, invalid \"fmt \" chunk size found (< 40)")', examples/tag_reader.rs:9:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/pcmwaveformat-16bit-44100Hz-mono-extra.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/pcmwaveformat-16bit-44100Hz-mono.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/pcmwaveformat-24bit-4byte-48kHz-stereo.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/pcmwaveformat-8bit-44100Hz-mono.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/pop.wav file..
thread 'main' panicked at 'Error: Failed to read file!: Wav("Extensible format identified, invalid \"fmt \" chunk size found (< 40)")', examples/tag_reader.rs:9:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/waveformatex-16bit-44100Hz-mono-extra.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/waveformatex-16bit-44100Hz-mono.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/waveformatex-16bit-44100Hz-stereo.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/waveformatex-8bit-11025Hz-mono.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/waveformatex-ieeefloat-44100Hz-mono.wav file..
thread 'main' panicked at 'attempt to divide by zero', src/iff/wav/properties.rs:165:31
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/waveformatextensible-24bit-192kHz-mono.wav file..
thread 'main' panicked at 'Error: Failed to read file!: Wav("Extensible format identified, invalid \"fmt \" chunk size found (< 40)")', examples/tag_reader.rs:9:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/waveformatextensible-24bit-4byte-48kHz-stereo.wav file..
thread 'main' panicked at 'Error: Failed to read file!: Wav("Extensible format identified, invalid \"fmt \" chunk size found (< 40)")', examples/tag_reader.rs:9:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/waveformatextensible-32bit-48kHz-stereo.wav file..
thread 'main' panicked at 'Error: Failed to read file!: Wav("Extensible format identified, invalid \"fmt \" chunk size found (< 40)")', examples/tag_reader.rs:9:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
------------Processing /home/user/Prejemi/hound-master/testsamples/waveformatextensible-ieeefloat-44100Hz-mono.wav file..
thread 'main' panicked at 'Error: Failed to read file!: Wav("Extensible format identified, invalid \"fmt \" chunk size found (< 40)")', examples/tag_reader.rs:9:10
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

### PR DESCRIPTION
**Pull request by sagudev**
and added test cases from hound.

Fixes #32.

### CODE REVIEW

File: tests/hound.rs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	for path in paths {
+		let path = path.unwrap().path();
+		if path.is_file() && path.extension().unwrap() == "wav" {
+			println!("Name: {}", path.display());
+			if let Ok(wav_reader) = WavReader::open(&path) {
+				let lofty = get_properties(&path).unwrap();
+				println!("{lofty:#?}");
+				assert_eq!(lofty.channels() as u16, wav_reader.spec().channels);
+				assert_eq!(lofty.sample_rate(), wav_reader.spec().sample_rate);
+				assert_eq!(lofty.bit_depth() as u16, wav_reader.spec().bits_per_sample);
```
Comment (Serial-ATA) [inline tests/hound.rs:45]: I tested this out, and this is never reached. Is it necessary?

File: src/iff/wav/properties.rs [hunk 2]
```diff
@@ -161,15 +170,18 @@ pub(super) fn read_properties(
...
-		let overall_bitrate = ((file_length * 8) / length) as u32;
-		let audio_bitrate = (u64::from(stream_len * 8) / length) as u32;
-
-		(
-			Duration::from_millis(length),
-			overall_bitrate,
-			audio_bitrate,
-		)
+		if length == 0 {
+			(Duration::from_secs(0), 0, 0)
```
Comment (Serial-ATA) [inline src/iff/wav/properties.rs:174]:
```suggestion
			(Duration::ZERO, 0, 0)
```

**Review by Serial-ATA — CHANGES_REQUESTED** [review]

File: tests/hound.rs (hunk 1 above)
Comment (sagudev) [reply tests/hound.rs:45]: This is made for any future syncs with hounds test files, that might not have same result.

Comment (Serial-ATA) [reply tests/hound.rs:45]: Ah, that makes sense.

**Review by Serial-ATA — APPROVED** [review]

**Comment by Serial-ATA** [comment]
Thanks!

**Pull request merged**